packages/shared/profiles/sagas.ts

```diff
@@ -78,11 +84,19 @@ export function* profileSaga(): any {
   yield takeEvery(USER_AUTHENTIFIED, initialRemoteProfileLoad)
   yield takeLatestByUserId(PROFILE_REQUEST, handleFetchProfile)
   yield takeLatestByUserId(PROFILE_SUCCESS, forwardProfileToRenderer)
+  yield takeEvery(PROFILES_REQUEST, handleFetchProfiles)
+  yield takeEvery(PROFILES_SUCCESS, forwardProfilesToRenderer)
   yield fork(handleCommsProfile)
   yield debounce(200, DEPLOY_PROFILE_REQUEST, handleDeployProfile)
   yield takeEvery(SAVE_PROFILE, handleSaveLocalAvatar)
 }
 
+function* forwardProfilesToRenderer(action: ProfilesSuccessAction) {
+  for (const profile of action.payload.profiles) {
+    yield put(sendProfileToRenderer(profile.userId))
+  }
+}
```

Comment:
This should be centralized in one single place, I'd prevent duplicating this action if it was not moved from its original location

Reply:
This is not being used, it was copied and pasted for an explanation during a call, it's going to be erased